bridge/core/dart_isolate_context.cc

```diff
@@ -187,10 +186,19 @@ void* DartIsolateContext::AddNewPage(double thread_identity,
   return nullptr;
 }
 
+std::unique_ptr<WebFPage> DartIsolateContext::InitializeNewPageSync(DartIsolateContext* dart_isolate_context,
+                                                                    size_t sync_buffer_size,
+                                                                    double page_context_id) {
+  dart_isolate_context->profiler()->StartTrackInitialize();
+  DartIsolateContext::InitializeJSRuntime();
```

Comment:
InitializeJSRuntime every times for creating new Page?

It's not correct. 

DartIsolateContext only initialized once when Dart env initialized, but Page could be initialized for multiple time for multiple WebF instance.